### ISSUE TITLE
Accessibility label for menu missing

### ISSUE THREAD

**Issue by WestphalDenn** [bigbluebutton/greenlight#2547]
In the current Version of Greenlight I have noticed that using a Screenreader, the Link to open the Menu where the Options for  Room Changes or adding a presentation are found doesn't have a Label added to it. So the Screenreader just reports "Link reduced"

It would help a lot if that link would be named $Room Menu" or something along those lines. That way blind users are sure what happens if we activate this link. 

Thanks in advance for making accessibility better.

**Comment by hadicheaito1** [comment]
Hello! Unfortunately, we will not be able to devote time into this feature/enhancement request at the moment due to our attention being directed towards Greenlight v3!
If this is something you believe that will be beneficial to v3, then we encourage you to please re-create this request once v3 is up and running.

To find out more information on Greenlight v3, please click the following link:
https://groups.google.com/g/bigbluebutton-dev/c/1nG-k_iTHjo

Thanks for your continued support!
